johnsnowlabs/utils/env_utils.py

```diff
@@ -104,12 +104,15 @@ def is_running_in_emr():
 
 
 def is_running_in_databricks():
-    """Check if the currently running Python Process is running in Databricks or not
-    If any Environment Variable name contains 'DATABRICKS' this will return True, otherwise False
+    """ Check if the currently running Python Process is running in Databricks or not
     """
-    for k in os.environ.keys():
-        if "DATABRICKS" in k:
-            return True
+    if "IS_IN_DATABRICKS_MODEL_SERVING_ENV" in os.environ:
+        # Serving container installs apache/spark, not databricks runtime.
+        return False
```

Comment:
Do we need this?

Reply:
+1, would prefer not to expose the specific env variable names we have in Databricks model serving.